### ISSUE TITLE
A11y_AzureDataStudio_ManageAccessForFile_ScreenReader:NVDA announcing the non interactive label as clickble.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7802]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#Win32;#NVDA;

**Environment Details: **

Application Name: Azure Data Studio

Version: 1.12.0-insider (user setup)

Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7

Date: 2019-10-03T08:56:27.454Z

VS Code: 1.38.0

Electron: 4.2.10

Chrome: 69.0.3497.128

Node.js: 10.11.0

V8: 6.9.427.31-electron.0

OS: Windows_NT x64 10.0.18362

 

**Additional Details:**

Tool Name : NVDA 2019.2
**Steps to Reproduce:**

1. Install and Launch the application the azure data studio application.

2.Connet to the servers with the required credentials.

3.Navigate to the control panel and click on the “Jupyter books” icon. 

 4.Navigate till HDFS folder and right click on it and click on manage access.

5. Start NVDA and observe the announcement.

**Actual:**
NVDA is announcing the non interactive element as clickable. For example  label and headings.  

 

**Expected:** 
NVDA should announce all non interactive in simple text format not clickable.  
 

**User Impact:**
Screen Reader user will misunderstand if the content is also interactive. 

**Attachment for Reference:**
[7802_A11y_AzureDataStudio_ManageAccessForFile_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3762502/7802_A11y_AzureDataStudio_ManageAccessForFile_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by V-sheman** [comment]
Verified the issue in Version: 1.15.0-insider (user setup)
The issue is resolved, Hence closing this bug
![image](https://user-images.githubusercontent.com/56344899/73456073-3bc35080-4397-11ea-885e-019d263f93a3.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
